src/drake/fs.clj

```diff
@@ -16,6 +16,19 @@
 (def drake-ignore "Names of files or directories to be ignored by Drake"
   #{"_logs"})
 
```

Comment:
This function's implementation is bizarre. Why is there a loop at all? Either the passed-in file is a directory, or its parent is. There's no use recurring. Additionally, the `(nil? f)` clause can never be true, because the previous clause will already have thrown an NPE if `f` is nil.